studio/components/interfaces/Auth/RedirectUrls/RedirectUrlList.tsx

```diff
@@ -13,15 +14,14 @@ interface RedirectUrlListProps {
 }
 
 const RedirectUrlList = ({ canUpdate, onSelectUrlToDelete = noop }: RedirectUrlListProps) => {
-  const { authConfig } = useStore()
+  const { ref: projectRef } = useParams()
+  const { data: config, isFetched: isLoaded } = useAuthConfigQuery({ projectRef })
```

Comment:
use isSuccess, add isLoading and isError states

Reply:
Done.